plugins/woocommerce/templates/emails/email-downloads.php

```diff
@@ -43,7 +43,7 @@ class="td font-family<?php echo $email_improvements_enabled ? ' email-order-deta
 	<?php foreach ( $downloads as $download ) : ?>
 		<tr>
 			<?php foreach ( $columns as $column_id => $column_name ) : ?>
-				<td class="td <?php echo $email_improvements_enabled && array_key_last( $columns ) === $column_id ? 'text-align-right' : 'text-align-left'; ?>">
+				<<?php echo 'download-product' === $column_id ? 'th scope="row"' : 'td'; ?> class="td <?php echo $email_improvements_enabled && array_key_last( $columns ) === $column_id ? 'text-align-right' : 'text-align-left'; ?>">
```

Comment:
I don't really like this, however the code needs something like this to account for the `woocommerce_email_downloads_columns` filter adding or removing columns.

Reply:
Yeah, agree that this is not ideal, as it also breaks highlighting or linting in IDEs. Even if more verbose and a bit duplicated, I'd go with 
```php
	<?php if ( 'download-product' === $column_id ) : ?>
		<th scope="row" class="td <?php echo $email_improvements_enabled && array_key_last( $columns ) === $column_id ? 'text-align-right' : 'text-align-left'; ?>">
	<?php else : ?>
		<td class="td <?php echo $email_improvements_enabled && array_key_last( $columns ) === $column_id ? 'text-align-right' : 'text-align-left'; ?>">
	<?php endif; ?>
```

and 
```php
	<?php if ( 'download-product' === $column_id ) : ?>
		</th>
	<?php else : ?>
		</td>
	<?php endif; ?>
```